collections/Hamilton/MS_Hamilton_49.xml

```diff
@@ -50,16 +43,41 @@
                   </altIdentifier>
                </msIdentifier>
                <physDesc>
-                  <p n="composite">Composite: two parts</p>
+                  <objectDesc form="codex">
+                     <supportDesc material="mixed">
+                        <support>paper and parchment (binding fragments)</support>
+                        <extent>362 + 4 (former pastedown) leaves <dimensions unit="mm" type="leaf">
+                              <height>210</height>
+                              <width>150</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Modern foliation.</foliation>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <binding notBefore="1459" notAfter="1500">
+                        <p>Contemporary, quarter-bound in leather over wooden boards with stamped fleur-de-lis designs. Paper labels on upper board, <q>Exposicio epistolarum dominicalium et officiorum misse per circulum anni</q> and <q>C 94</q>. Three clasps, now lacking.</p>
+                     </binding>
+                     <binding notBefore="1850" notAfter="1950">
+                        <p>Rebacked, nineteenth to mid-twentieth century.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
+               <history>
+                  <origin>
+                     <origDate calendar="Gregorian" notAfter="1475" notBefore="1450">15th century, 3rd quarter</origDate>
+                     <origPlace>
+                        <country key="place_7000084">German</country>, <settlement key="place_7024031">Erfurt</settlement>, <orgName role="fmo" key="org_415145858126223022295">Charterhouse of Salvatorberg</orgName>: in its medieval catalogue with the signature <q>C 94</q> (also labelled on the upper board), as printed by P. Lehmann, <title>MBDS</title> ii. 232–593. </origPlace>
+                  </origin>
```

Comment:
I think the Erfurt Charterhouse should have a `provenance` entry in addition and the library catalogue information should probably go there.

Reply:
I think I have now done enough to show that Erfurt can be treated as the origin, but let me know if I am missing something.